doc/client.rst

```diff
@@ -587,6 +587,38 @@ Single-Record Transactions
     .. index::
         single: User Defined Functions
 
+Multi-Record Transactions
```

Comment:
I think the `get_status` argument functionality should be explicitly documented for both MRT methods if it isn't split into its own APIs.